pkg/v1/providers/config_default.yaml

```diff
@@ -547,13 +547,14 @@ AVI_DISABLE_INGRESS_CLASS: true
 AVI_AKO_IMAGE_PULL_POLICY: IfNotPresent
 AVI_ADMIN_CREDENTIAL_NAME: avi-controller-credentials
 AVI_CA_NAME: avi-controller-ca
+AVI_CONTROLLER_VERSION: 20.1.3
 
 AVI_CONTROLLER:
 AVI_USERNAME: ""
 AVI_PASSWORD: ""
 AVI_CLOUD_NAME:
 AVI_SERVICE_ENGINE_GROUP:
-AVI_CONTROLLER_VERSION: 20.1.3
+AVI_NSXT_T1LR:
```

Comment:
It is not very obvious what is supposed to be supplied for this value, how about adding a comment here?